content/docs/started/getting-started-k8s.md

```diff
@@ -1,524 +1,30 @@
 +++
-title = "Kubeflow on Kubernetes"
-description = "Instructions for installing Kubeflow on your existing Kubernetes cluster"
+title = "Kubeflow on Existing Kubernetes Clusters"
+description = "Instructions for installing Kubeflow on your existing Kubernetes cluster with list of supported options"
 weight = 4
 +++
 
 Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster.
 
 If you are using a Kubernetes distribution or Cloud Provider which has specific instructions for installing Kubeflow we recommend following those instructions. Those instructions do additional Cloud specific setup to create a really great Kubeflow experience.
 
-For installing Kubeflow on an existing Kubernetes Cluster, you can use one of the following options:
+The following table lists the options for installing Kubeflow on an existing Kubernetes Cluster and links to detailed instructions.
 
-* [Kubeflow for Existing Clusters by Arrikto](#Kubeflow-for-Existing-Clusters---by-Arrikto)
+### Community maintained
 
-## Kubeflow for Existing Clusters - by Arrikto
+This section includes vendor-neutral solutions governed by community consensus. Below deployment configs are maintained and supported by the community.
 
-This installation of Kubeflow is maintained by [Arrikto](https://www.arrikto.com/), is geared towards existing Kubernetes clusters and does not depend on any cloud-specific feature.
 
-In this reference architecture, we use [Dex](https://github.com/dexidp/dex) and [Istio](https://istio.io/) for vendor-neutral authentication.
+|  Deployment config|      Description      |
```

Comment:
For code readability and better table rendering, use HTML instead of Markdown for tables.

Example:
```
<div class="table-responsive">
  <table class="table table-bordered">
    <thead class="thead-light">
      <tr>
        <th>My col header 1</th>
        <th>My col header 2</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>My content row 1 col 1</td>
        <td>My content row 1 col 2</td>
      </tr>
      <tr>
        <td>My content row 2 col 1</td>
        <td>My content row 2 col 2</td>
      </tr>
    </tbody>
  </table>
</div>
```

See [this page](https://www.kubeflow.org/docs/about/style-guide/#use-simple-present-tense) for an example of how the tables render on the page.